src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -1138,6 +1141,26 @@
   <script src="{{ asset('../js/tiny_mce/tiny_mce.js') }}"></script>
   <script src="{{ asset('../js/admin/tinymce.inc.js') }}"></script>
   <script src="{{ asset('../js/admin/tinymce_loader.js') }}"></script>
+
+  <script>
+      $(function() {
+        var editable = '{{ editable }}'
```

Comment:
If I want use a twig variable in javascrip i must define him like that : '{{ editable }}'.

I modify the condition